templates/configmap.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
```

Comment:
Do we need this configmap ? Seems we already have an `initFile` variable that I'm assuming does almost close to the same thing ?

Reply:
we do not, I'll remove it